### PR TITLE
+readme Cross link to extras

### PR DESCRIPTION
**Pull request by ktoso**
crosslink to extras

resolves https://github.com/apple/swift-metrics/issues/87

### CODE REVIEW

**Review by Lukasa — APPROVED** [review]

**Comment by ktoso** [comment]
Cory seems to not have approval powers :<

Can I ask for a thumbs up @tomerd @yim-lee ?

File: README.md [hunk 1]
```diff
@@ -58,6 +58,12 @@ As the API has just launched, not many implementations exist yet. If you are int
 - [StatsD Client](https://github.com/apple/swift-statsd-client), support for StatsD
 - Your library? [Get in touch!](https://forums.swift.org/c/server)
 
+### Swift Metrics Extras
+
+You may also be interestedin some "extras" modules which are collected in the [swift metrics extras](https://github.com/apple/swift-metrics-extras) repository.
```
Comment (yim-lee) [inline README.md:63]:
```suggestion
You may also be interestedin some "extras" modules which are collected in the [Swift Metrics Extras](https://github.com/apple/swift-metrics-extras) repository.
```

?

File: README.md [hunk 2]
```diff
@@ -58,6 +58,12 @@ As the API has just launched, not many implementations exist yet. If you are int
...
+
+Currently the repository includes: `SystemMetrics` allowing reporting of a node's CPU and memory metrics etc.
```
Comment (yim-lee) [inline README.md:65]:
```suggestion
Currently the repository includes `SystemMetrics` which allows reporting of a node's CPU and memory metrics etc.
```

**Review by yim-lee — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (tomerd) [inline README.md:63]:
```suggestion
You may also be interested in some "extras" modules which are collected in the [swift metrics extras](https://github.com/apple/swift-metrics-extras) repository.
```

File: README.md (hunk 2 above)
Comment (tomerd) [reply README.md:65]: not sure its worth detailing what extra includes as it won't scale to update both places. maybe just mention the kind of things that extra may include.

Comment (ktoso) [reply README.md:65]: Yeah that's a concern -- I'll just link to it then

File: README.md (hunk 1 above)
Comment (ktoso) [reply README.md:63]: Thanks :)

File: README.md (hunk 2 above)
Comment (ktoso) [inline README.md:65]:
```suggestion
```

File: README.md [hunk 3]
```diff
@@ -58,6 +58,11 @@ As the API has just launched, not many implementations exist yet. If you are int
 - [StatsD Client](https://github.com/apple/swift-statsd-client), support for StatsD
 - Your library? [Get in touch!](https://forums.swift.org/c/server)
 
+### Swift Metrics Extras
+
+You may also be interestedin some "extras" modules which are collected in the [Swift Metrics Extras](https://github.com/apple/swift-metrics-extras) repository.
```
Comment (ktoso) [inline README.md:63]:
```suggestion
You may also be interested in some "extra" modules which are collected in the [Swift Metrics Extras](https://github.com/apple/swift-metrics-extras) repository.
```

**Comment by ktoso** [comment]
Thanks for review, made it just a link rather than mention what it has

**Pull request merged**